src/sentry/integrations/analytics.py

```diff
@@ -13,6 +13,16 @@ class IntegrationAddedEvent(analytics.Event):
     )
 
 
+class IntegrationDisabledNotified(analytics.Event):
+    type = "integration.disabled.notified"
+
+    attributes = (
+        analytics.Attribute("organization_id"),
```

Comment:
@chloeho7 can we add a user id so we can send it to Amplitude? It can be the default owner id.